lensless/utils/io.py

```diff
@@ -379,6 +379,8 @@ def load_data(
     psf_fp,
     data_fp,
     background_fp=None,
+    return_bg=False,
+    legacy_remove=True,
```

Comment:
Instead of "legacy_remove" I think the namer "remove_background" is better?

Because from what I understand, you want to load (psf, data, background) without removing background from data.

Also there's some loading logic that needs to be fixed. Because if you're returning the background, you will also want to normalize it similar to how the data is being normalized (like [here](https://github.com/LCAV/LenslessPiCam/blob/0424e15966eaaf7d7a5c04d5e69d14297f92b32f/lensless/utils/io.py#L522) also dividing by `data.max()` to make sure they are scaled by the same amount). Otherwise your background will not be in the same range. Maybe that's what gave you issues before?